src/main/config/config_streamer.c

```diff
@@ -53,6 +53,8 @@ extern uint8_t __config_end;
 #  define FLASH_PAGE_SIZE                 ((uint32_t)0x8000)
 # elif defined(UNIT_TEST)
 #  define FLASH_PAGE_SIZE                 (0x400)
+# elif defined(SITL)
```

Comment:
For me 'self explanatory' means you don't have to google it. But that might just be me.